api/core/model_runtime/model_providers/openai_api_compatible/llm/llm.py

```diff
@@ -367,13 +367,16 @@ def create_final_llm_result_chunk(index: int, message: AssistantPromptMessage, f
 
         for chunk in response.iter_lines(decode_unicode=True, delimiter=delimiter):
             if chunk:
+                #ignore ": OPENROUTER PROCESSING" to support openrouter.ai streaming
+                chunk = chunk.strip().lstrip(': OPENROUTER PROCESSING').lstrip()                
+            if chunk:                
```

Comment:
This is a deprecate if statement. Please move the `chunk = chunk.strip().lstrip(': OPENROUTER PROCESSING').lstrip()` into next line.